package/MDAnalysis/core/universe.py

```diff
@@ -465,7 +465,7 @@ def transfer_to_memory(self, start=None, stop=None, step=None, verbose=None, qui
             # fall back to a slower approach
             except AttributeError:
                 pm = ProgressMeter(self.trajectory.n_frames,
-                                   interval=step, verbose=verbose)
+                                   interval=1, verbose=verbose)
```

Comment:
The correct fix would be to calculate the number of frames based on `start`, `stop`, `step` and an interval in percent to keep in from updating all the time. So the interval can be calculated like this.

```python
interval = int(n_frames // 100)
if interval == 0:
    interval = 1
```